README.md

```diff
@@ -1,66 +1,66 @@
 # AWSLambda
 
-The helper library to invoke [AWS Lambda](http://docs.aws.amazon.com/lambda) functions from agent code.
+This library can be used to invoke [Amazon Web Services (AWS) Lambda](http://docs.aws.amazon.com/lambda) functions from your agent code. It depends upon the [AWSRequestV4](https://github.com/electricimp/AWSRequestV4/) library, which **must** be included.
 
-To add this library to your model, add the following lines to the top of your agent code:
+**To add this library to your code, add**
 
 ```
-#require "AWSRequestV4.class.nut:1.0.2"
 #require "AWSLambda.agent.lib.nut:1.0.0"
+#require "AWSRequestV4.class.nut:1.0.2"
```

Comment:
That's a good point, @ppetrosh. I changed the order to emphasize the library the documentation describes. As @betzrhodes says, it's not an issue here, but it might be, so probably worth adding '#require order' to the list of tests you do in review. I think the named library should always be listed first unless there's a specific need that another one comes first.